semantic-source/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.1.0.2
+
+- Support ghc 9.2.
+
```

Comment:
I bumped `semantic-source`, which turned out to be rather confusing, but eh, it's out there now.